tests/test_client.py

```diff
@@ -1,6 +1,13 @@
-import pytest
+import os
 import concurrent
-from unittest.mock import ANY
+from unittest.mock import ANY, patch
+
+import pytest
+from google.cloud.pubsub_v1 import SubscriberClient
+
+from rele.client import Subscriber
+
+from . import settings
```

Comment:
You should use `django.conf.settings` object

Reply:
As far as I know, we want to decouple the library from Django as far as possible. That said, and seeing that many other test modules apply the same pattern, I think we import settings that way to avoid importing `django`.

Does that make sense to you? I'm not 100% sure and want to better understand the case 😄

Reply:
Actually I think we can remove that settings, we just use it to get the project_id and the credentials but now it's passed as parameter. I'll work to remove that.